lib/tasks/update_application.rake

```diff
@@ -82,6 +82,19 @@ class UpdateApplicationRakeTask
           logger.info("Application #{application.ecf_id} cohort changed from #{cohort_before_update} to #{application.cohort.start_year}")
         end
       end
+
+      desc "Change the schedule on an application"
+      task :update_schedule, %i[application_ecf_id new_schedule_identifier] => :environment do |_t, args|
+        find_application(args.application_ecf_id)
+
+        current_schedule = application.schedule
+
+        new_schedule = Schedule.find_by(identifier: args.new_schedule_identifier, cohort: application.cohort)
+        raise "Schedule not found: #{args.new_schedule_identifier}" unless new_schedule
+
+        application.update!(schedule: new_schedule)
```

Comment:
Since schedules impact declarations I think this should be checking the application has no declarations?